docs/examples/single_point_io.rst

```diff
@@ -10,7 +10,13 @@ To adapt this to Frames, just change the sessions to
 :any:`nixnet.session.FrameOutSinglePointSession` with frames instead of
 signals.  Then adjust ``read``/``write`` to take a frame object per frame
 configured in the session rather than signals.  This works for both CAN and LIN
-frames.  See :doc:`queued_io` to see how to read and write frames.
+frames. LIN frames also require ``change_lin_sched`` to write a request for the
+LIN interface to change the running schedule  See :doc:`queued_io` to see how
+to read and write frames. See :doc:`can_lin_diff` to see how to convert a CAN
+example into a LIN example.
```

Comment:
Is it sufficient to point them to `queued_io` and rely on that example pointing to the document "how to adapt CAN to LIN"?